apps/prairielearn/src/lib/config.ts

```diff
@@ -356,7 +356,7 @@ export const ConfigSchema = z.object({
    * logs to cheaper storage tiers or evict them entirely after a certain
    * amount of time.
    */
-  workspaceLogsExpirationDays: z.number().default(120),
+  workspaceLogsExpirationDays: z.number().default(120).nullable(),
```

Comment:
Nit, the `nullable()` conventionally comes before the `default()`:

```suggestion
  workspaceLogsExpirationDays: z.number().nullable().default(120),
```